fonts.css

```diff
@@ -0,0 +1,15 @@
+@font-face {
+  font-family: "GothamPro";
+  src: 
+    url("../fonts/GothamPro.woff") format("woff"),
+    url("fonts/GothamPro.woff2") format("woff2");
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: "GothamPro-Bold";
+  src: 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5542092)